modules/migration-creating-migration-plan-cam.adoc

```diff
@@ -15,7 +15,7 @@ You can create a migration plan in the {mtc-full} ({mtc-short}) web console.
 * You must ensure that the same {mtc-short} version is installed on all clusters.
 * You must add the clusters and the replication repository to the {mtc-short} web console.
 * If you want to use the _move_ data copy method to migrate a persistent volume (PV), the source and target clusters must have uninterrupted network access to the remote volume.
-* If you want to use direct image migration, the `MigCluster` custom resource manifest of the source cluster must specify the exposed route of the internal image registry.
+* If you want to use direct image migration, you must specify the exposed route to the image registry of the source cluster by using the {mtc-short} web console or by updating the `MigCluster` custom resource manifest.
```

Comment:
Consider splitting into two smaller sentences:

```suggestion
* If you want to use direct image migration, you must specify the exposed route to the image registry of the source cluster. This can be done by using the {mtc-short} web console or by updating the `MigCluster` custom resource manifest.
```